test/Microsoft.Azure.Devices.Edge.Test/Metrics.cs

```diff
@@ -45,7 +43,9 @@ class Report
 
         void BaseConfig(EdgeConfigBuilder builder)
         {
-            builder.AddModule(this.moduleName, this.imageName);
+            Assert.IsTrue(Context.Current.MetricsValidatorImage.HasValue, "Metrics validator image must be provided in context.json");
+
+            builder.AddModule(this.moduleName, Context.Current.MetricsValidatorImage.OrDefault());
```

Comment:
The tests use the following pattern for required test parameters (instead of Assert):

```
string metricsValidatorImage = Context.Current.MetricsValidatorImage.Expect(() => new InvalidOperationException("Missing Metrics Validator image"));

builder.AddModule(this.moduleName, metricsValidatorImage);
```